packages/styling/src/styles/fonts.ts

```diff
@@ -107,25 +107,19 @@ function _fontFamilyWithFallbacks(fontFamily: string): string {
 export function createFontStyles(localeCode: string | null): IFontStyles {
   const localizedFont = _getLocalizedFontFamily(localeCode);
   let fontFamilyWithFallback = _fontFamilyWithFallbacks(localizedFont);
-  let semilightFontFamilyWithFallback = fontFamilyWithFallback;
-
-  // Chrome has a bug where it does not render Segoe UI Semilight correctly, so we force the webfont to be used in that case
-  if (localizedFont === defaultFontFamily) {
```

Comment:
@christiango this part was added by you due to some issues of Chrome with semi light font weight of Segoe UI font. Now we are not using it anymore so I removed this logic. Let me know if I need to make any extra changes anywhere else because this is new territory to me and don't wanna mess things up 😊 . Thanks!